.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+_[Put a one line description of your change into the PR title, please be specific]_
+
+_[Explain the context, and why you're making that change. What is the problem you're trying to solve.]_
+
+-[Tests can be run by commenting @swift-ci test on the pull request]
```

Comment:
Should we link to the testing section of the README here, for more info? And we can fill in more detail there over time

Reply:
Should the `@swift-ci test` part be backtick-quoted?